homework/index.js

```diff
@@ -0,0 +1,30 @@
+const promise = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(response => response.json());
+
+function selectRepo() {
+  promise.then((data) => {
+    data.forEach((i) => {
+      const repolist = document.createElement('option');
+      const opt = document.createTextNode(i.name);
+      repolist.appendChild(opt);
+      document.getElementById('repoList').appendChild(repolist);
```

Comment:
Similarly, a `<select>` element has an `add` method which allows you to directly add `option`s.